src/api-reference/travel/itinerary/trip/trip-resource.markdown

```diff
@@ -353,44 +366,43 @@ The response returns subset of the elements described in the following tables de
 | id	|string	|Trip ID URI with encrypted ID.
 |ItinLocator | string	|The itinerary locator. This element is now deprecated and only supported for backward compatibility.
 |ClientLocator	|string	|The XSD does not have a description for this element. Please provide one.
```

Comment:
Whoops. ClientLocator has reviewer notes in it again. Let's remove the existing description and send a query to the Travel PM.

Reply:
You sent the query the last time. I don't have the contact info.